src/fiskaltrust.AndroidLauncher.Http/Controllers/ITSSCDController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using fiskaltrust.ifPOS.v1.it;
+using Microsoft.AspNetCore.Mvc;
+using System.ServiceModel;
+using System.Threading.Tasks;
+using static Java.Util.Jar.Attributes;
+
+namespace fiskaltrust.AndroidLauncher.Http.Controllers
```

Comment:
We shouldn't forget to add @StefanKert's latest crimes in the IITSSCD here before merging this.